Codeinwp/neve
# Fix comments section

### Summary
- Fixed the display of the comments section
- Added visual regression for the comments section
- Improved the e2e tests with the same changes I previously did on Neve Pro

### Will affect the visual aspect of the product
NO

### Test instructions
- Enable comments for posts
- Write a long comment and a short one
- Check the alignment of the reply button


Closes #3884.
